rl_games/common/a2c_common.py

```diff
@@ -620,7 +620,10 @@ def set_weights(self, weights):
     def _preproc_obs(self, obs_batch):
         if type(obs_batch) is dict:
             for k,v in obs_batch.items():
-                obs_batch[k] = self._preproc_obs(v)
+                if v.dtype == torch.uint8:
```

Comment:
should it go to the else block in the _preproc_obs call with tensor any way?

Reply:
No, since it causes a crash if `normalise_input=True` (it expects `obs_batch` to be a dict if dict obs are used).